docs/mux-logging-service.md

```diff
@@ -49,6 +49,20 @@ There are environment variables for the logging-mux DC config:
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
+These are environment variables for the logging-fluentd daemonset config:
+
+* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
```

Comment:
Definitely for 3.7, but the only way forward we see for scaling is thru mux.  I fear that this is going to open up more problems to describe to the field how to get it working.

If we focus our effort only on one solution that we know works, wouldn't we have a better time supporting that instead of having to support so many variations?